docs/src/languages/php/tuning.md

```diff
@@ -8,211 +8,186 @@ Platform.sh offers a wide degree of flexibility in how PHP behaves,
-so send it good ones.
+The default worker count is set to a conservative default value.
+To determine and set the optimal value for your app, see [PHP-FPM sizing](./fpm.md).
 
-Static assets cache headers are set using the `expires` key in `.platform.app.yaml`.
-See the [`web.locations` documentation](../../create-apps/app-reference.md#locations) for more details.
+## OPcache preloading
 
-## Optimize the FPM worker count
+OPcache preloading loads selected files into shared memory, making their content (functions, classes) globally available for requests.
+It also removes the need to include these files later.
+When OPcache is correctly configured, it can result in significant improvements to both CPU and memory usage.
 
-PHP-FPM reserves a fixed number of simultaneous worker processes to handle incoming requests.
-If more simultaneous requests are received than the number of workers,
-then some requests wait.
-The default worker count is deliberately set rather conservative
-but can be improved in many cases.
-See the [PHP-FPM sizing](./fpm.md) page
-for how to determine and set a more optimal value.
+Consult your framework's documentation to see
+if there are recommendations for optimal preload configuration or ready-to-use preload scripts.
 
-## Enable preloading
+OPcache is only available on PHP 7.4+ and uses PHP-CGI.
+If your PHP version doesn't support OPcache, this is a good reason to upgrade.
 
-PHP 7.4 and later supports preloading code files into shared memory once at server startup,
-bypassing the need to include or autoload them later.
-Depending on your application doing so can result in significant improvements
-to both CPU and memory usage.
-If using PHP 7.4 or later, see the [PHP Preload instructions](./_index.md#opcache-preloading)
-for how to configure it on Platform.sh.
-Consult your application's documentation to see
-if they have any recommendations for an optimal preload configuration.
+Note that the only way to clear the preload cache is by [restarting PHP-FPM](#restart-php-fpm).
 
-If you aren't using PHP 7.4, this is a good reason to upgrade.
+### Enable OPcache
 
-Note that the only way to clear the preload cache is by restarting PHP-FPM.
-PHP-FPM isn't restarted on every deployment automatically,
-so you might want to add that in a [`deploy` hook](../../create-apps/hooks/hooks-comparison.md#deploy-hook),
-such as by including `pkill -f php-fpm` or `sv restart app`.
+To enable preloading, add a variable that specifies a preload script:
 
-If you have [disabled OPcache timestamp validation](#disable-opcache-timestamp-validation),
-you need to clear the OPcache explicitly on deployment (which can be done by restarting PHP-FPM).
+```yaml {location=".platform.app.yaml"}
+variables:
+    php:
+        opcache.preload: '<PRELOAD_SCRIPT>'
+```
 
-## Configure OPcache
+The `opcache.preload` value is a file path relative to the [app root](../../create-apps/app-reference.md#root-directory).
```

Comment:
This kind of explanation might help in places above (like where there was `PATH_TO_FILE`